tests/Fixture/TestContainer.ts

```diff
@@ -15,6 +16,20 @@ export class TestContainer {
         this.container = new Container(this.id, this.params);
     }
 
+    /**
+     * Adds a shape drawer to the current container. Drawers do not persist and will be removed
+     * when `TestContainer.reset` is called.
+     *
+     * @param type
+     * @param drawer
+     */
+    public addShapeDrawer(type: string, drawer: IShapeDrawer): void {
```

Comment:
This should be unnecessary, in a recent commit I needed something like this because the emitters had a bug in 1.14.0 changes, we could try without this method so we can test that fix too